app/models/changelog_data.php

```diff
@@ -46,9 +47,119 @@
     '3.8.1' => '2015-06-24',
     '3.9'   => '2015-10-08',
     '3.10'  => '2015-11-13',
+    '4.0'   => '2016-01-04',
 ];
 
 $changelog = [
+    '4.0' => [
+        [
+            'message' => ['Propose search suggestions as your type in the search field'],
+            'section' => ['end_user'],
+            'type'    => ['new'],
+            'issues'  => [551],
+            'authors' => ['pascal', 'flod'],
+        ],
+        [
+            'message' => ['Create a <a href="/rss/">RSS</a> feed for release notes'],
+            'section' => ['end_user'],
+            'type'    => ['new'],
+            'issues'  => [200],
+            'authors' => ['tchevalier'],
+        ],
+        [
+            'message' => ['Remove unused "wildcard" search option'],
+            'section' => ['end_user'],
+            'type'    => ['better'],
+            'issues'  => [549],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Update results count when filtering search results by component'],
+            'section' => ['end_user'],
+            'type'    => ['bug'],
+            'commit'  => ['f26b5fb58047ba8c460ff6bd41dabb461b18a91d'],
+            'authors' => ['flod'],
+        ],
+        [
+            'message' => ['Stop hard-coding version information in the application.'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'commit'  => ['b7dbe66c4837ef557a40c57ee5baab86dff2d353'],
+            'authors' => ['flod'],
+        ],
+        [
+            'message' => ['Update Travis CI configuration toexclude xdebug and add PHP 5.6 and 7 to the testing matrix'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'issues'  => [553],
+            'authors' => ['SkySymbol'],
+        ],
+        [
+            'message' => ['Update Verif, our functional testing library (nicer output)'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'commit'  => ['a500d77a3478b305b5cd82beb5a09e30a9fe228d'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Move all our regular expression logic into a new unit tested class named <var>Search</var>'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'issues'  => [320],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Transvision moved to a faster server with hardware redundancy sponsored by <a href="https://www.gandi.net/">Gandi</a>'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Upgraded OS from Debian 7 to 8. Transvision minimum PHP requirement moved from 5.4 to 5.6'],
```

Comment:
I see you updated the Wiki, but we will need to update the readme file as well :) https://github.com/mozfr/transvision#dependencies